src/menus/icon-menu.jsx

```diff
@@ -114,12 +108,6 @@ const IconMenu = React.createClass({
      * horizontal: [left, center, right].
      */
     targetOrigin: PropTypes.origin,
-
-    /**
-     * Sets the delay in milliseconds before closing the
-     * menu when an item is clicked.
-     */
-    touchTapCloseDelay: React.PropTypes.number,
```

Comment:
Maybe to allow time to see the ripple before the menu closes? (Pure speculation.)